uint8 id_x_left
uint8 id_x_right
uint8 id_y_left
uint8 id_y_right
int32 x_position
int32 y_position
